gems/aws-sigv4/lib/aws-sigv4/signer.rb

```diff
@@ -693,8 +712,135 @@ def credentials_set?(credentials)
           !credentials.secret_access_key.empty?
       end
 
+      ### CRT Code
+
+      # the credentials used by CRT must be a
+      # CRT StaticCredentialsProvider object
+      def fetch_credentials_crt
```

Comment:
I think crt_ should be a prefix and not a suffix (at least when inspecting methods it's more obvious)

Reply:
Sure - I don't have a preference as these are private methods.

Reply:
(even for public ones too)